lib/intercom-rails/script_tag.rb

```diff
@@ -26,6 +27,19 @@ def initialize(options = {})
       self.nonce = options[:nonce]
     end
 
+    def remove_user_cookie_on_logout
+      if (find_current_user_details == {} && controller.response.request.cookies["intercom-session-#{IntercomRails.config.app_id}"])
+        controller.response.set_cookie("intercom-session-#{IntercomRails.config.app_id}", :value => "", :expires => Time.at(0))
```

Comment:
If this is a `Rack::Response`, can we use `delete_cookie`?

http://www.rubydoc.info/github/rack/rack/Rack/Response/Helpers#delete_cookie-instance_method